db-services/db-diagram-service/src/main/java/com/qreal/wmp/db/diagram/server/DiagramDbServiceHandler.java

```diff
@@ -100,4 +101,19 @@ public TFolder getFolderTree(String username) throws TNotFound {
         }
         return folder.toTFolder();
     }
+
+    /** Only for sake of testing. */
+    public DiagramDao getDiagramDao() {
```

Comment:
I don't see any other option (excepting moving tests for server to server package)